package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "on-chain-identity-gateway",
+    "version": "1.0.0",
+    "description": "Welcome to the Identity.com On-chain Identity Gateway monorepo.",
+    "main": "index.js",
+    "directories": {
+        "doc": "docs"
+    },
+    "devDependencies": {
+        "husky": "^8.0.1",
+        "npm-run-all": "^4.1.5"
+    },
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "prepare": "husky install",
+        "lint": "cd solana/program_v2 && yarn lint"
+    },
+    "husky": {
+        "hooks": {
+        "pre-commit": "cargo fmt --manifest-path solana/program_v2/Cargo.toml --all -- --check && yarn lint"
```

Comment:
This seems duplicate to `.husky/pre-commit` I'm sure that does not need to be necessary?

Reply:
yeah, you were right, I deleted this duplication.